### PR TITLE
fix(scripts/run_test.sh): Creating missing .config directory

### PR DESCRIPTION
**Pull request by Orenef11**

### CODE REVIEW

File: scripts/run_test.sh [hunk 1]
```diff
@@ -54,9 +54,17 @@ export JMX_DIR=${JMX_DIR:-`pwd`/../scylla-jmx}
 export CCM_DIR=${CCM_DIR:-`pwd`/../scylla-ccm}
 export SCYLLA_DBUILD_SO_DIR=${SCYLLA_DBUILD_SO_DIR:-${CASSANDRA_DIR}/dynamic_libs}
 
-mkdir -p ${HOME}/.ccm
-mkdir -p ${HOME}/.m2
-mkdir -p ${HOME}/.local/lib
+if [[ ! -d ${HOME}/.ccm ]]; then
```
Comment (fruch) [inline scripts/run_test.sh:57]: no need for those IF, that's exactly what `mkdir -p` is doing

File: scripts/run_test.sh [hunk 2]
```diff
@@ -54,9 +54,17 @@ export JMX_DIR=${JMX_DIR:-`pwd`/../scylla-jmx}
...
+  mkdir -p ${HOME}/.ccm
+fi
+if [[ ! -d ${HOME}/.m2 ]]; then
+  mkdir -p ${HOME}/.m2
+fi
+if [[ ! -d ${HOME}/.local/lib ]]; then
+  mkdir -p ${HOME}/.local/lib
+fi
+if [[ ! -d ${HOME}/.config ]]; then
+  mkdir -p ${HOME}/.config
```
Comment (fruch) [inline scripts/run_test.sh:67]: you are missing a closing `fi` here

File: scripts/run_test.sh (hunk 1 above)
Comment (Orenef11) [reply scripts/run_test.sh]: thanks!

**Comment by fruch** [comment]
what about mounting this directory ? otherwise I don't think this alone would fix anything

**Comment by Orenef11** [comment]
> what about mounting this directory

I compared it against python-driver-matrix code, and the bash file install the same directories.
Which directory do you think is missing?
https://github.com/scylladb/python-driver-matrix/blob/master/scripts/run_test.sh#L38-L43

**Comment by fruch** [comment]
> > what about mounting this directory
> 
> I compared it against python-driver-matrix code, and the bash file install the same directories. Which directory do you think is missing? https://github.com/scylladb/python-driver-matrix/blob/master/scripts/run_test.sh#L38-L43

In the python matrix later the code runs in virtualenv (it's a bit different), also it's not used from the ccm commandline as here.
I would compare to things in dtest run_test.sh, and there `.local` is being mounted

**Comment by Orenef11** [comment]
@fruch
I've checked it and I think the job is working now

**Comment by Orenef11** [comment]
@fruch 
The PR is ready for review

**Review by roydahan — APPROVED** [review]

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**